### ISSUE TITLE
latest released version not available on conda-forge

### ISSUE THREAD

**Issue by yunjunz** [Jupyter-contrib/jupyter_nbextensions_configurator#167]
The latest version on GitHub and PyPI is 0.6.3 from 3 months ago, while the latest version on conda-forge is 0.6.1 from ~8 months ago. May I ask if is there a reason for this difference? I am using conda-forge more often and would appreciate it if the conda-forge channel has the latest fixes as well. Many thanks!

**Comment by TarekMarrouhMoreno** [comment]
Exactly, the version that is in conda is 6.1 while in master 6.3 is used.
https://anaconda.org/conda-forge/jupyter_nbextensions_configurator

**Comment by saiwing-yeung** [comment]
Wonder if there are any updates? Thanks.
